modules/trainer/GenericTrainer.py

```diff
@@ -89,6 +90,8 @@ def start(self):
             torch.backends.cuda.matmul.allow_tf32 = True
             torch.backends.cudnn.allow_tf32 = True
 
+        ZLUDA.setup(self.config)
```

Comment:
This line is only executed when training. That means it won't work for other tools like the sampling UI. Can you also move it to `scripts/util/import_util.py`, to call the initialization for every script?

Reply:
done.